auraloss/freq.py

```diff
@@ -209,7 +215,7 @@ def stft(self, x):
     def forward(self, input: torch.Tensor, target: torch.Tensor):
         bs, chs, seq_len = input.size()
 
-        if self.perceptual_weighting:  # apply optional A-weighting via FIR filter
+        if self.prefilter is not None:  # apply prefilter
```

Comment:
Here we can add two checks, one for `if self.prefilter is not None:` and one for `if self.user_prefilter is not None:`.